Qt/QtMain.cpp

```diff
@@ -159,11 +159,24 @@ int System_GetPropertyInt(SystemProperty prop) {
 #else
 		return DEVICE_TYPE_DESKTOP;
 #endif
+	case SYSPROP_DISPLAY_COUNT:
+		return QApplication::screens().size();
 	default:
 		return -1;
 	}
 }
 
+int System_GetPropertyFloat(SystemProperty prop) {
```

Comment:
Adding this probably affects headless and unittest as well.  It might be simpler to use Int and round to a certain precision (i.e. * 100.)  But I'm not really opposed to adding Float, as long as we add it all the places.

-[Unknown]

Reply:
I saw this usage for refresh rate. But I think it's kinda tricky. I'll add GetPropertyFloat all the places